jetstream/change-app-color-icon-android-for-existing-users-v2.toml

```diff
@@ -1,20 +1,26 @@
 
 [metrics]
 
-weekly = ["app_opened", "app_icon_used"]
+weekly = ["app_opened", "app_icon_used", "app_opens_per_client", "app_icon_uses_per_client"]
 
 # Metrics to compute for each 28 days long analysis window
-28_day = ["app_opened", "app_icon_used"]
+28_day = ["app_opened", "app_icon_used", "app_opens_per_client", "app_icon_uses_per_client"]
 
 # Metrics to compute only for the overall analysis window.
-overall = ["app_opened", "app_icon_used"]
+overall = ["app_opened", "app_icon_used", "app_opens_per_client", "app_icon_uses_per_client"]
 
 [metrics.app_opened]
 select_expression = "CAST(SUM(IF(event.name = 'app_opened',1,0)) >0 AS INT)"
 data_source = "events"
 friendly_name = "app opened"
 description = "app opened"
 
+[metrics.app_opens_per_client]
+select_expression = "SUM(IF(event.name = 'app_opened',1,0))"
```

Comment:
How should you handle a client who never opens the app... in that case they would never send an event. Right now, they would be represented as a `NULL` and wouldn't count toward the metric. Is this the intention? If you want them to count as a zero, you should wrap this in a `COALESCE(..., 0)`

Reply:
Thank you for pointing this out. I am updating both new metric definitions.